hashutils/id.go

```diff
@@ -33,7 +33,10 @@ func GetLengthLimitedID(fixedPrefix, suffix string, maxLength int) string {
 		// start with the character that we use to denote a shortened string, which could
 		// result in a clash.  Hash the value and truncate...
 		hasher := sha256.New()
-		hasher.Write([]byte(suffix))
+		_, err := hasher.Write([]byte(suffix))
+		if err != nil {
+			return ""
```

Comment:
Should probably panic here.  I think hashers are contracted not to return errors (and returning "" doesn't handle the error)

Reply:
Yeah, I wasn't sure what to do here. Thanks.
Fixed